src/Components/Filtering/Multiselect.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import { Typography } from '@mui/material';
+// import Button from '@mui/material/Button';
+import propTypes from 'prop-types';
+import MultiSelectComponent from './MultiSelectComponent';
+
+const filters = [
+  {
+    filterName: 'Ethnicity',
+    filterOptions: ['Black/African American', 'White', 'Asian', 'Hawaiian/Pacific Islander', 'Hispanic',
+      'American Indian/Alaska Native', 'Indian', 'American'],
+  },
+  {
+    filterName: 'Religion',
+    filterOptions: ['Christian', 'Muslim', 'Hindu', 'Jewish', 'Atheist'],
+  },
+  {
+    filterName: 'Gender',
+    filterOptions: ['Male', 'Female', 'Nonbinary', 'Other'],
+  },
+  {
+    filterName: 'Sexuality',
+    filterOptions: ['Gay', 'Straight', 'Bisexual', 'Other'],
+  },
+];
+
+export default function MultSelectElem({ setTempMultiSelect, tempMultiSelect }) {
```

Comment:
Function names should always match the name of the file they are in!

Reply:
also can you match the names tempMultiSelect and setTempMultiSelect with the labels in the component below input and setInput? Either of the name combos are fine, just pick the one you prefer.

Reply:
I think you haven't yet updated the name!